tests/sdk/api/test_wf_run.py

```diff
@@ -1366,7 +1370,7 @@ def test_run(
         monkeypatch.setattr(_api._config.RuntimeConfig, "name", "auto")
         with raises:
             wf_def.run("in_process", workspace_id=workspace_id, project_id=project_id)
-            workflow_create_mock.assert_called_once_with(wf_def.model, expected)
+            workflow_create_mock.assert_called_once_with(wf_def.model, expected, False)
```

Comment:
nitpick: please use named arguments when passing literals.

Reply:
For all the comments about passing literals: I can do changes with function call, cant apply them to mock.asserts - if parameter is passed as arg, it has to be checked as arg, not kwarg